### PR TITLE
docs: Refresh docs for SQL input source

### PR DESCRIPTION
**Pull request by vtlim**
Updates docs for the SQL input source, MySQL extension, and PostgreSQL extension for usability and style.

This PR has:

- [x] been self-reviewed.

### CODE REVIEW

File: docs/development/extensions-core/druid-lookups.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ To use this Apache Druid extension, [include](../../configuration/extensions.md#
 :::info
  If using JDBC, you will need to add your database's client JAR files to the extension's directory.
```
Comment (techdocsmith) [inline docs/development/extensions-core/druid-lookups.md:34]:
```suggestion
To use JDBC, you must add your database client JAR files to the extension's directory.
```
nit: avoid future tense

File: docs/development/extensions-core/mysql.md [hunk 2]
```diff
@@ -25,41 +25,58 @@ title: "MySQL Metadata Store"
 
 To use this Apache Druid extension, [include](../../configuration/extensions.md#loading-extensions) `mysql-metadata-storage` in the extensions load list.
 
-:::info
- The MySQL extension requires the MySQL Connector/J library or MariaDB Connector/J library, neither of which are included in the Druid distribution.
- Refer to the following section for instructions on how to install this library.
-:::
+The MySQL extension lets you use MySQL as a metadata store or ingest from a MySQL database.
```
Comment (techdocsmith) [inline docs/development/extensions-core/mysql.md:28]:
```suggestion
With the MySQL extension, you can use MySQL as a metadata store or ingest from a MySQL database.
```

File: docs/development/extensions-core/mysql.md [hunk 3]
```diff
@@ -90,7 +107,7 @@ This extension also supports using MariaDB server, https://mariadb.org/download/
   CREATE DATABASE druid DEFAULT CHARACTER SET utf8mb4;
 
   -- create a druid user
-  CREATE USER 'druid'@'localhost' IDENTIFIED BY 'diurd';
+  CREATE USER 'druid'@'localhost' IDENTIFIED BY 'druid';
```
Comment (techdocsmith) [inline docs/development/extensions-core/mysql.md:110]: FWIW i think `diurd`  is an intentional misspelling for the password.

File: docs/development/extensions-core/postgresql.md [hunk 4]
```diff
@@ -25,7 +25,9 @@ title: "PostgreSQL Metadata Store"
 
 To use this Apache Druid extension, [include](../../configuration/extensions.md#loading-extensions) `postgresql-metadata-storage` in the extensions load list.
 
-## Setting up PostgreSQL
+The PostgreSQL extension lets you use PostgreSQL as a metadata store or ingest from a PostgreSQL database.
```
Comment (techdocsmith) [inline docs/development/extensions-core/postgresql.md:28]:
```suggestion
With the  PostgreSQL extension, you can use PostgreSQL as a metadata store or ingest from a PostgreSQL database.
```

File: docs/ingestion/input-sources.md [hunk 5]
```diff
@@ -833,17 +825,29 @@ For more information on the `maxNumConcurrentSubTasks` field, see [Implementatio
...
+* To connect to MySQL, load the extension [`mysql-metadata-storage`](../development/extensions-core/mysql.md).
+* To connect to PostgreSQL, load the extension [`postgresql-metadata-storage`](../development/extensions-core/postgresql.md).
+
+The MySQL extension requires a JDBC driver.
+For more information, see the [Installing the MySQL connector library](../development/extensions-core/mysql.md).
+:::
+
 The SQL input source is used to read data directly from RDBMS.
-The SQL input source is _splittable_ and can be used by the [Parallel task](./native-batch.md), where each worker task will read from one SQL query from the list of queries.
+The SQL input source is _splittable_ and can be used by the [parallel task](./native-batch.md), where each worker task will read from one SQL query from the list of queries.
```
Comment (techdocsmith) [inline docs/ingestion/input-sources.md:838]:
```suggestion
You can _split_ the ingestion tasks for a SQL input source. When you use the [parallel task](./native-batch.md) indexer?, each worker task reads from one SQL query from the list of queries.
```
Consider rewording for clarity.

File: docs/ingestion/input-sources.md [hunk 6]
```diff
@@ -887,7 +891,7 @@ Compared to the other native batch input sources, SQL input source behaves diffe
 
 The Combining input source lets you read data from multiple input sources.
 It identifies the splits from delegate input sources and uses a worker task to process each split.
-Use the Combining input source only if all the delegates are splittable and can be used by the [Parallel task](./native-batch.md).
+Use the Combining input source only if all the delegates are splittable and can be used by the [parallel task](./native-batch.md).
```
Comment (techdocsmith) [inline docs/ingestion/input-sources.md:894]: this reads awkwardly 😬  I know you were just fixing case.

**Review by techdocsmith — CHANGES_REQUESTED** [review]
minor comments, mostly style

File: docs/ingestion/input-sources.md (hunk 6 above)
Comment (vtlim) [reply docs/ingestion/input-sources.md:894]:
```suggestion
Each delegate input source must be splittable and compatible with the [parallel task type](./native-batch.md).
```

**Review by techdocsmith — APPROVED** [review]
LGTM

**Review by zachjsh — APPROVED** [review]

**Pull request merged**